MainWindow.h

```diff
@@ -108,11 +114,15 @@ class MainWindow : public QMainWindow {
 
   std::unique_ptr<buffers::Project> _project;
   QPointer<RecentFiles> _recentFiles;
+  
+  std::unique_ptr<EventData> _event_data;
+  egm::EGM egm;

```

Comment:
Move all of this stuff to the private section.

Reply:
It is in private....